main.tf

```diff
@@ -285,6 +285,13 @@ resource "aws_iam_policy" "iam_policy" {
         "arn:aws:s3:::snowplow-hosted-assets-ap-northeast-2",
         "arn:aws:s3:::snowplow-hosted-assets-ap-northeast-2/*"
       ]
+    },
+    {
+      "Action": [
+        "cloudwatch:PutMetricData"
```

Comment:
Hi Ben, the cloudwatch metrics are disabled in the config.hocon, why do we need the extra permissions?

Reply:
When I tested it was showing permissions errors. But you're right it should not need them. Let me check